app/templates/surgery.html

```diff
@@ -0,0 +1,64 @@
+{% extends "base.html" %}
+
+
+{% block page_content %}
+    <div class="post-tabs">
+        <h4>Insert record of surgery.</h4>
+	<h5><b>Note:</b> Enter 0 for weight or ketoprofen if not measured.</h5>
```

Comment:
In the table you will insert em as NULL, I think is better if you get rid of the InputRequired validation in the form and ask users to leave em blank. Or at least require user to set them as -1 (if so, set -1 as default).